keyboards/dztech/dz65rgb/v3/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = lufa-ms
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite        # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no          # Mouse keys
+EXTRAKEY_ENABLE = yes          # Audio control and System control
+CONSOLE_ENABLE = no            # Console for debug
+COMMAND_ENABLE = no            # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no          # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes              # USB Nkey Rollover
+BACKLIGHT_ENABLE = no          # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no           # Enable keyboard RGB underglow
+MIDI_ENABLE = no               # MIDI support
+BLUETOOTH_ENABLE = no          # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no              # Audio output on port C6
+FAUXCLICKY_ENABLE = no         # Use buzzer to emulate clicky switches
+RGB_MATRIX_ENABLE = yes        # Use RGB matrix
+RGB_MATRIX_DRIVER = IS31FL3741
+NO_USB_STARTUP_CHECK = no      # Disable initialization only when usb is plugged in
+WAIT_FOR_USB = yes
```

Comment:
No and this should get removed (I removed it in my kbdfans/dztech cleanup) since it is known to cause problems with the board not being detected after rebooting the computer. 
We had tons of people complain about this on the kbdfans discord. Removing that option solved it for all of them.